fn/option.go

```diff
@@ -61,14 +61,11 @@ func (o Option[A]) UnwrapOrFunc(f func() A) A {
 func (o Option[A]) UnwrapOrFail(t *testing.T) A {
 	t.Helper()
 
-	if o.isSome {
-		return o.some
+	if !o.isSome {
+		t.Fatalf("Option[%T] was None()", o.some)
```

Comment:
`require.False(o.isSome)`

Reply:
Won't that make the error message that appears in a test less helpful though?

Reply:
Require throws the line that failed so should be enough and it allows setting a message: 
`require.False(t, o.isSome, "Option %T is None()", o.some)`